ApiSpecifications.md

```diff
@@ -138,9 +138,6 @@ POST <partners/databricks/v1/connect>: [example, can be customized]
      "is_connection_established" : true|false
      "auth": { [Only present if is_connection_established is false]
        "personal_access_token": "dapi..."
```

Comment:
Just to verify my understanding, in the implementation, we don't actually need to pass this oauth information?

Reply:
So I see with current M2M implementation, we don't need these fields.  Will we ever need them for U2M?

Reply:
no, we will not need these fields in either M2M or U2M.